### PR TITLE
feat: add login and signup form

### PR DESCRIPTION
**Pull request by AtulPant2704**
- CSB Link for Login:- https://vberu.csb.app/authentication/login/login.html
- CSB Link for Signup:- https://vberu.csb.app/authentication/sign-up/sign-up.html

### CODE REVIEW

File: authentication/sign-up/sign-up.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script defer src="https://kit.fontawesome.com/829cb68a23.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://checkmate-ui.netlify.app/css/components.css" />
+    <link rel="stylesheet" href="./sign-up.css">
+    <title>LogIn</title>
```
Comment (starver20) [inline authentication/sign-up/sign-up.html]: Change to signup

Comment (starver20) [reply authentication/sign-up/sign-up.html]: Also input box size is different on both pages. Everything else looks good.

**Review by mohammed-sas — APPROVED** [review]
pr good to merge

**Review by kart-c — APPROVED** [review]
Apart from the issues mentioned above by @starver20 everything else looks good

**Review by starver20 — APPROVED** [review]

**Pull request merged**